_lessons/python-scatter/01b-profiling-map.md

```diff
@@ -38,11 +38,13 @@ cd mpi
 
 ## Using the "Express Launch"
 
-To use MAP we need to load the *forge* module in our batch script and add `map --profile` in front of the parallel run statements. For example:
+To use MAP we need to load the *forge* module in our batch script and add `map --profile` in front of the executable statements. For example:
 ```
 module load forge
-map --profile srun python scatter.py
+map --profile map python scatter.py
```

Comment:
There is a typo. "map --profile map python scatter.py" should be "map --profile srun python scatter.py". Thanks for fixing.